frontend/src/scenes/notebooks/Nodes/NodeWrapper.tsx

```diff
@@ -252,6 +257,19 @@ export function createPostHogWidgetNode<T extends CustomNotebookNodeAttributes>(
         atom: true,
         draggable: true,
 
+        serializedText: serializedText,
```

Comment:
Would something like 
```suggestion
        serializedText: () => { serializedText() },
```
to save you from doing the double function in each of the node files? I think that syntax might be confusing for people

Reply:
The problem is that serializedText is called with no args when we're creating the nodes in our extendSchema function. 

So you either have to enforce that `serializedText` can be called with undefined as well as attrs or curry the function like this (or something I'm not thinking of 🙈)

Reply:
Ha, so I can simply not call it 🤦